monitoring/ping.go

```diff
@@ -230,7 +230,7 @@ func (c *pingChecker) saveLatencyStats(pingLatency int64, node serf.Member) erro
 
 	s, exists := c.latencyStats.Get(node.Name)
 	if !exists {
-		sMap = make([]int64, slidingWindowSize)
+		sMap = make([]int64, 0, slidingWindowSize)
```

Comment:
why is this slice called `sMap`?

Reply:
initial it was actually a `map` then since a Slice is just a map between int and something else the name remained. Anyway I fixed it now.

Reply:
@eldios That's interesting way to think about a slice, although I'm afraid incorrect :) Slice is backed by array and array and map are two completely different data structures with different properties.